doc/visual-programming/source/loading-your-data/index.rst

```diff
@@ -133,45 +118,41 @@ This is how the header with augmented attribute names looks like in
 Excel (:download:`sample-head.xlsx <sample-head.xlsx>`):
 
 .. image:: spreadsheet-simple-head1.png
-    :width: 500 px
+    :width: 414 px
     :align: center
 
-We can again use a `Data Table <http://docs.orange.biolab.si/widgets/rst/data/datatable.html#data-table>`_ widget to read the data from Excel file. Orange will automatically recognize attribute values, which is evident in the modified class icons:
+We can again use again a **File** widget to load this data set and then render it in the **Data Table**:
```

Comment:
Dataset [skipped](https://www.google.com/trends/explore#q=%22data%20set%22%2C%20%22dataset%22%2C%20%22data-set%22&cmpt=q&tz=Etc%2FGMT-1) the hyphenated stage. :stuck_out_tongue: